eq-author/src/App/publish/PublishPage.js

```diff
@@ -88,61 +131,84 @@ const PublishPage = ({ match, history }) => {
     <Container>
       <Header title="Publish" />
       <ScrollPane>
-        <StyledGrid>
-          <AlignedColumn cols={6}>
-            <Field>
-              <Label htmlFor="surveyId">Survey ID</Label>
-              <Caption>Enter survey ID</Caption>
-              <Shadow>
-                <Input
-                  id="surveyId"
-                  onChange={handleInputChange}
-                  value={inputs.surveyId}
+        <StyledPanel>
+          <Field>
+            <Label>Theme</Label>
```

Comment:
```suggestion
            <Label>Themes</Label>
```

Design is slightly different for this.